lib/src/main/java/uk/gov/di/ipv/core/library/service/AuditService.java

```diff
@@ -24,21 +25,24 @@ public static AmazonSQS getDefaultSqsClient() {
         return AmazonSQSClientBuilder.defaultClient();
     }
 
-    public void sendAuditEvent(AuditEventTypes eventType) throws SqsException {
+    public void sendAuditEvent(AuditEventTypes eventType, AuditExtensionParams extensions)
```

Comment:
Could you create a new method signature that replicates the old one, that just calls this with `null` for extensions? That way you don't have to call this method with `null` in places you're not passing in the extensions?

Reply:
Yeah good shout, I'll update it